src/lib/Microsoft.Fx.Portability/ApiPortClient.cs

```diff
@@ -179,6 +145,22 @@ public ApiPortClient(IApiPortService apiPortService, IProgressReporter progressR
             }
         }
 
+        private Dictionary<string, ResultFormatInformation> AvailableReportFormats
```

Comment:
I'm not a fan of this async call being made synchronous. Can we leverage AsyncLazy<T> instead?